requirements.txt

```diff
@@ -1,9 +1,9 @@
 dj-database-url==0.3.0
 Django==1.8.13
-django-debug-toolbar==1.3.2
 django-jinja==1.4.1
 django-jinja-bootstrap-form==4.0.3
 funcy==1.7.2
+django-model-utils==1.3.1
```

Comment:
If we upgrade django-pandas to 0.4 there will no need of django-model-utils

It seems that we will need to upgrade to Django 1.9
https://github.com/chrisdev/django-pandas/search?q=model_utils&type=Issues&utf8=%E2%9C%93